x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/details_page/components/package_policies/package_policies_table.tsx

```diff
@@ -109,29 +116,62 @@ export const PackagePoliciesTable: React.FunctionComponent<Props> = ({
           defaultMessage: 'Name',
         }),
         render: (value: string, packagePolicy: InMemoryPackagePolicy) => (
-          <EuiLink
-            title={value}
-            {...(canReadIntegrationPolicies
-              ? {
-                  href: getHref('edit_integration', {
-                    policyId: agentPolicy.id,
-                    packagePolicyId: packagePolicy.id,
-                  }),
-                }
-              : { disabled: true })}
-          >
-            <span className="eui-textTruncate" title={value}>
-              {value}
-            </span>
-            {packagePolicy.description ? (
-              <span>
-                &nbsp;
-                <EuiToolTip content={packagePolicy.description}>
-                  <EuiIcon type="help" />
-                </EuiToolTip>
-              </span>
-            ) : null}
-          </EuiLink>
+          <EuiFlexGroup gutterSize="s" alignItems="center">
+            <EuiFlexItem data-test-subj="PackagePoliciesTableName" grow={false}>
+              <EuiLink
+                title={value}
+                {...(canReadIntegrationPolicies
+                  ? {
+                      href: getHref('edit_integration', {
+                        policyId: agentPolicy.id,
+                        packagePolicyId: packagePolicy.id,
+                      }),
+                    }
+                  : { disabled: true })}
+              >
+                <span className="eui-textTruncate" title={value}>
+                  {value}
+                </span>
+                {packagePolicy.description ? (
```

Comment:
can you add a screenshot of how this looks if an integration policy has description text?

Reply:
Sure, I forgot to add it. The description is added on an icon tooltip:

![Screenshot 2024-07-02 at 09 56 01](https://github.com/elastic/kibana/assets/16084106/d259009c-9403-418b-a366-4d56555270b7)

![Screenshot 2024-07-02 at 09 56 06](https://github.com/elastic/kibana/assets/16084106/14cda014-49c9-43da-851c-291145058f60)